clients/src/test/java/org/apache/kafka/common/log/remote/storage/LocalTieredStorage.java

```diff
@@ -112,15 +169,20 @@ public void configure(Map<String, ?> configs) {
         if (transfererClass != null) {
             try {
                 transferer = (Transferer) getClass().getClassLoader().loadClass(transfererClass).newInstance();
+
             } catch (InstantiationException | IllegalAccessException | ClassNotFoundException | ClassCastException e) {
                 throw new RuntimeException(format("Cannot create transferer from class '%s'", transfererClass), e);
             }
         }
 
-        Directory directory = openDirectory(ROOT_STORAGES_DIR_NAME + "/" + storageId, new File("."));
-        storageDirectory = directory.directory;
+        storageDirectory = new File(new File("."), ROOT_STORAGES_DIR_NAME + "/" + storageId);
```

Comment:
Good to have a config for parent directory, it should default to a temp dir if it is not configured.